### ISSUE TITLE
Handle Follow redirect 

### ISSUE THREAD

**Issue by thibaultleouay** [openstatusHQ/openstatus#810]
### Description

Right now the default is we follow the redirect 
we should add an option to the monitor configuration ( boolean column) to follow or not the redirect

**Comment by kotiyalashwin** [comment]
So basically a new column can be added to the monitor table name Redirect and create corresponding zod schemas. As of exploring i find this point where change is to be made. Please respond if my approach is right for now.

> Also the default value of redirect should be True or False?

![Image](https://github.com/user-attachments/assets/e466f965-944a-4b8f-bf6f-b4a457e0485b)

**Comment by thibaultleouay** [comment]
@kotiyalashwin  Yep we could add a new column followRedirect column and update all the related zod schema 
by default it's true 

For the frond end it could be done after

**Comment by kotiyalashwin** [comment]
> [!NOTE]
> Please tell if this is all that was required. 

I have changed the schema of the table. Now every new monitor entry will have a  `followRedirect` attribute whose value is set to `true` or `1` by default

![Image](https://github.com/user-attachments/assets/78240339-e430-442d-a171-698c4946437f)


>[!NOTE]
**Any changes to be done in frontend or this much is fine for now.**

> Frontend receiving monitor `followRedirect` attribute successfully. 

**overview/page.tsx**

![Image](https://github.com/user-attachments/assets/791bc327-4fb8-4102-9489-60c11a44a193)

**logged data**

![Image](https://github.com/user-attachments/assets/88462f07-7459-4167-a68b-31999387b911)

**Comment by thibaultleouay** [comment]
@kotiyalashwin  if you want you can take inspiration from 
https://github.com/openstatusHQ/openstatus/pull/1261
